### DISCUSSION TITLE
'loadable' util for mutations

### DISCUSSION THREAD

**Discussion by rusakovic** [pmndrs/jotai#2843] [Q&A]
Hello.

Can  https://jotai.org/docs/utilities/async  `loadable` util be used with mutation functions?

Also in docs
```
const asyncAtom = atom(async (get) => ...)
```

the example is not correct, should be 'null' as a minimum for `initialValue: null`

**Comment by dai-shi** [comment]
We make utils small and primitive. Our suggestion is to create another derived atom.

See also: https://github.com/pmndrs/jotai/discussions/2792 https://github.com/pmndrs/jotai/discussions/1061#discussioncomment-2406494

As I said in #2792, there's a possibility though. I just feel it seems confusing with `loadable`.

> the example is not correct

It is incomplete but correct. It's an async derived atom. Maybe, making it complete would help.
